### ISSUE TITLE
Please add version or Product name to exe properties on NZBGet!

### ISSUE THREAD

**Issue by ng4ever** [nzbget-ng/nzbget#16]
For some reason the original developer never wanted to and I have no idea why. :(

Someone else requested it too.

Main reason is to keep up to date with it on SUMo updater.

Here is my previous topic for it: https://github.com/nzbget/nzbget/issues/759

**Comment by paul-chambers** [comment]
Makes sense to do so, but I'm not versed in Win32 and don't know how to do it.

This is one of the reasons I'd like to find a few other maintainers to cover stuff I'm unfamiliar with.

**Comment by ng4ever** [comment]
Ok no problem!

**Comment by salami-ch** [comment]
I'm currently experimenting with the Windows build environment and adding a version resource looks like a [fairly small change](https://github.com/salami-ch/nzbget-win64/commit/386a75c8b5cf7fab08b43d9fc61d5da9e6eb6f9c). Unfortunately I'm also far from an expert and I don't want to pollute this fork just yet.

**Comment by paul-chambers** [comment]
Looks like @salami-ch is working on it: https://github.com/salami-ch/nzbget-win64/

Hopefully there will be a pull request once he's happy with it. In the meantime, he's got a release you could try?
